day_7/projects/caeser_cipher.rb

```diff
@@ -0,0 +1,30 @@
+class CaesarCipher
+  attr_accessor :position
```

Comment:
As you're only reading the `@position` attribute, you'd only need an `attr_reader` for it. Rule of thumb is don't expose writing capabilities unless you absolutely need to.